src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -404,6 +404,11 @@ public void missingMapping(RegistryEvent.MissingMappings<Item> event)
                 ResourceLocation newTotem = new ResourceLocation("minecraft:totem_of_undying");
                 entry.remap(ForgeRegistries.ITEMS.getValue(newTotem));
             }
+            if (entry.key.toString().equals("forge:bucket_filled"))
```

Comment:
The entry is not going to equal `forge:bucket_filled`, the event is called with the missing ID `forge:bucketfilled` (it is registered as `bucketFilled` but gets lowercased down the line)